MapboxCoreNavigation/String.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
```

Comment:
@frederoni you think this okay? I can't use `UIDevice.current.systemName` without it. You think hardcoding strings would be okay here?